### DISCUSSION TITLE
Searx not showing search page as front page

### DISCUSSION THREAD

**Discussion by yodahome** [searx/searx#1178] [Installation issue: custom]
I installed Searx on my Ubuntu 16.04 vServer with uWSGI and Apache2 as described here: https://asciimoo.github.io/searx/dev/install/installation.html

However when opening the URL https://searx.art3mis.de/ I get an error message and not the search page. https://searx.art3mis.de/preferences works and https://searx.art3mis.de/search works also, but you can't search for anything, because it only takes you to the front page (the error). 

The only thing I did differently is adding the options for apache2 to a virtual host file for the subdomain and not to apache2.conf directly, but I don't see how this could lead to an error. If I access it directly on port 8888 it works as expected. It seems like some bad rewriting is going on, but I can't for the life of me figure out how to fix it. 

Any hints would be appreciated, I'm sure it's a dumb mistake on my part.

Edit:

This is what the vHost looks like

```
<IfModule mod_ssl.c>
<VirtualHost *:443>
        ServerAdmin webmaster@art3mis.de
        ServerName searx.art3mis.de
        <Location />
            Options +FollowSymLinks +Indexes
            SetHandler uwsgi-handler
            uWSGISocket /run/uwsgi/app/searx/socket
        </Location>
SSLCertificateFile /etc/letsencrypt/live/???/fullchain.pem
SSLCertificateKeyFile /etc/letsencrypt/live/???/privkey.pem
Include /etc/letsencrypt/options-ssl-apache.conf
</VirtualHost>
</IfModule>
```

**Comment by Pofilo** [comment]
We may need to see the configuration of the vhost to help. Can you provide it ?

**Comment by yodahome** [comment]
Of course! Sorry, I forgot and updated the original post with the vHost.

**Comment by ghost** [comment]
Is there anything new, I have the same here. How to setup the searchsite as default?

**Comment by ghost** [comment]
found the solution here
https://github.com/asciimoo/searx/issues/686#issuecomment-469916026
